packages/calling-stateful-client/src/TestUtils.ts

```diff
@@ -95,15 +94,15 @@ export class MockRecordingCallFeatureImpl implements RecordingCallFeature {
   public recordings;
   public emitter = new EventEmitter();
   on(event: 'isRecordingActiveChanged', listener: PropertyChangedEvent): void;
-  /* @conditional-compile-remove(calling-1.4.3-beta.1) */
-  on(event: 'recordingsUpdated', listener: CollectionUpdatedEvent<RecordingInfo>): void;
+  // TODO: revert to using `RecordingInfo`.
```

Comment:
Working with the calling team to  be able to undo this before this PR is merged.